lib/chewy/errors.rb

```diff
@@ -37,6 +37,6 @@ def initialize(join_field_type, join_field_name, relations)
     end
   end
 
-  class ImportScopeCleanupError < Error
```

Comment:
This ImportScopeCleanError was in this branch committed by Ben. Is this something we need? It isn't present in the current working gem that we use.